dubesar/Ultimate-Java-Resources
# Ivanebernal/memegenerator

## What does this PR do?

Adds meme generator app

## Test Plan

- Use an image from gallery
- Modify text
- Modify text size
- Modify text color
- Save image
- Check gallery for new image

## Related PRs and Issues
Closes #398 

### Have you read the [Contributing Guidelines](https://github.com/dubesar/Java-A-Z/blob/master/CONTRIBUTING.md)?

Yes
